deployments/liqo/values.yaml

```diff
@@ -90,6 +90,8 @@ gateway:
     portOverride: ""
     # -- port used by the vpn tunnel.
     listeningPort: 5871
+    # -- implementation used by wireguard. Possible values are "userspace" and "kernel".
```

Comment:
I would add a warning to the user to not use the userspace version unless necessary